fs/xfstests.py.data/xfs/v4.yaml

```diff
@@ -0,0 +1,17 @@
+scratch_mnt: '/mnt/scratch'
+test_mnt: '/mnt/test'
+disk_mnt: '/mnt/loop-device'
+
+loop_type: !mux
+    type: 'loop'
+    loop_size: '12GiB'
+    # Option to provide disk for loop device creation,
+    # Uses '/' by default for file creation
+    disk: "null"
+
+fs_type: !mux
+    fs_xfs_v4:
+        fs: 'xfs'
+        args: '-R xunit -L 10 -g quick'
+        mkfs_opt: '-f -m crc=0'
```

Comment:
Let's be explicit about the blocksize here? (default 4096 maybe)?
And make the file name as 4k_v4.yaml.